src/redux/reducers/backendStatus/index.js

```diff
@@ -21,6 +25,18 @@ const backendStatusReducer = (state = initialState, action) => {
     case BACKEND_STATUS_ERROR: {
       return backendStatusError(state, action);
     }
+    case BACKEND_STATUS_DELETED: {
+      return backendStatusDeleted(state, action);
+    }
+    case HYDRATE: {
```

Comment:
Out of ignorance, what is this used for?

Reply:
these are changes introduced by Marcell from this PR -https://github.com/biomage-ltd/ui/pull/499